### PR TITLE
Dependencies upgrade

### PR DESCRIPTION
**Pull request by vyobukhov**
### Updated
- **BREAKING** Minimum Node.js version required is 10
- Upgraded dependencies
	- Upgraded stylelint to v13
	- Upgraded postcss-query-ast to v2
### Internal
- Replaced Mocha tests with Jest

#2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,21 +14,21 @@
...
     "release": "np",
-    "test": "eslint '{index,lib/**/*,test/**/*}.js' && mocha --require esm 'test/*.js' --delay",
+    "test": "jest --silent",
     "version": "version-changelog CHANGELOG.md && changelog-verify CHANGELOG.md && git add CHANGELOG.md"
   },
   "dependencies": {
-    "postcss-query-ast": "^1.0.0",
-    "stylelint": "^9.9.0"
+    "postcss-query-ast": "^2.0.0",
+    "stylelint": "^13.13.1"
```
Comment (niksy) [inline package.json:26]: @vyobukhov this could be peer dependancy since Stylelint plugin can’t work without Stylelint anyways

File: package.json [hunk 2]
```diff
@@ -60,5 +61,11 @@
   "bugs": {
     "url": "https://github.com/niksy/stylelint-no-restricted-syntax/issues"
   },
-  "homepage": "https://github.com/niksy/stylelint-no-restricted-syntax#readme"
+  "homepage": "https://github.com/niksy/stylelint-no-restricted-syntax#readme",
+  "jest": {
```
Comment (niksy) [inline package.json:65]: @vyobukhov why do we need Jest? I don’t recall that Stylelint forces testing library. See how I did it in one of my [recent plugins](https://github.com/niksy/stylelint-no-unresolved-module/blob/master/test/util/index.js)

**Review by niksy — CHANGES_REQUESTED** [review]

**Comment by niksy** [comment]
@vyobukhov Thanks for the changes! I’ve added some comments so please review them before we can merge this.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "stylelint-no-restricted-syntax",
-  "version": "1.0.0",
+  "version": "2.0.0",
```
Comment (niksy) [inline package.json:3]: @vyobukhov this will be changed automatically with release process

**Review by niksy — CHANGES_REQUESTED** [review]

**Comment by igoradamenko** [comment]
@niksy, hey there!

Would you mind to merge and release this PR? 

Or maybe I can help you somehow with this process?

**Comment by niksy** [comment]
@igoradamenko this is already done in latest version so I’m going to close this PR.

**Comment by igoradamenko** [comment]
Ah, lol, you're right. I haven't checked the latest version prior to writing this comment 🤦 

Thank you!

**Pull request closed without merging**